[
  {
    "id" : "4ffd6770-d406-4d6f-bc9c-46bd2447678c",
    "prId" : 10861,
    "prUrl" : "https://github.com/lampepfl/dotty/pull/10861#pullrequestreview-558423501",
    "prSource" : "GitHub",
    "comments" : [
      {
        "id" : "265b6298-7ee8-4881-a4fd-035dec2915e8",
        "parentId" : null,
        "authorId" : "6998ebc0-40b8-4e52-b4f8-721fd5d8c0a9",
        "body" : "Do we really want to allow this? I somehow parse this as:\r\n\r\n```scala\r\nif x < 0 then { throw();\r\n  val ex = ...\r\n}\r\n```",
        "createdAt" : "2020-12-23T11:56:31Z",
        "updatedAt" : "2021-01-01T12:35:24Z",
        "lastEditedBy" : "6998ebc0-40b8-4e52-b4f8-721fd5d8c0a9",
        "tags" : [
        ]
      },
      {
        "id" : "ae1a5710-7384-4b64-b4fc-7f165256ec77",
        "parentId" : "265b6298-7ee8-4881-a4fd-035dec2915e8",
        "authorId" : "d08d2c4a-dc79-4be6-96b0-506d21e90d1d",
        "body" : "For consistency I think it is better to allow it. The given expression abuses indentation rules for sure. It should be formatted like this:\r\n```\r\nif x < 0 then\r\n   throw\r\n      val ex = ...\r\n      ex\r\n```\r\n\r\n\r\n",
        "createdAt" : "2020-12-24T09:15:05Z",
        "updatedAt" : "2021-01-01T12:35:24Z",
        "lastEditedBy" : "d08d2c4a-dc79-4be6-96b0-506d21e90d1d",
        "tags" : [
        ]
      }
    ],
    "commit" : "625f7540a325e4d13781fdd2785d3308247e07b2",
    "line" : 17,
    "diffHunk" : "@@ -1,1 +86,90 @@  self =>\n  def foo(x: Int) =\n    if x < 0 then throw\n      val ex = new AssertionError()\n      ex"
  }
]